proto/colibri/v1/ratelimiter.proto

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/scionproto/scion/go/pkg/proto/colibri";
+
+package proto.colibri.v1;
+
+
+service RateLimiterService {
+    rpc AddRateLimit(AddRateLimitRequest) returns (Success) {}
+    rpc SetBurstSizeAndRate(SetBurstSizeAndRateRequest) returns (Success) {}
+    rpc SetBurstSize(SetBurstSizeRequest) returns (Success) {}
+    rpc SetRate(SetRateRequest) returns (Success) {}
+    
+
+}
+
+// /////////////////////////////////////////////////
+
+message AddRateLimitRequest {
+    string address = 1;
```

Comment:
I would suggest to put them in a more meaningful order.
e.g. first where to apply something (address, ingress, egress), and then what to do (cbs, rate).

Applies to all the Requests in this file.

Reply:
Yes that makes sense